build.gradle

```diff
@@ -106,3 +90,7 @@ spotless {
         bumpThisNumberIfACustomStepChanges(2)
     }
 }
+
+apply from: rootProject.file('buildscript/spotless-rules.gradle')
```

Comment:
shouldn't at least part of the 'patches' format move here too

Reply:
Probably not, this was isolating the part that was shared between `build.gradle`, `tests/build.gradle`, and `testframework/build.gradle` - the patches formatting only appears in `build.gradle`, and I'm not sure if there would be side effects of it existing in the subprojects